rspec-rails.gemspec

```diff
@@ -27,9 +27,9 @@ Gem::Specification.new do |s|
     s.cert_chain = [File.expand_path('~/.gem/rspec-gem-public_cert.pem')]
   end
 
-  s.add_runtime_dependency(%q<activesupport>, [">= 3.0"])
-  s.add_runtime_dependency(%q<actionpack>, [">= 3.0"])
-  s.add_runtime_dependency(%q<railties>, [">= 3.0"])
+  s.add_runtime_dependency(%q<activesupport>, [">= 3.0", "<= 4.2"])
+  s.add_runtime_dependency(%q<actionpack>, [">= 3.0", "<= 4.2"])
+  s.add_runtime_dependency(%q<railties>, [">= 3.0", "<= 4.2"])
```

Comment:
Recently, we've needed to add some updates for the new releases. I believe they have been most ERs, but one or two deprecations came up. I would feel safer locking this down to avoid the "I updated and it broke" scenario. This way our support is explicit and Bundler / Rubygems will inform people of mismatches ahead of time. See history in #1187 and #1192.